contracts/interfaces/IERC7160.sol

```diff
@@ -0,0 +1,60 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts v4.4.1 (token/ERC721/extensions/IERC721Metadata.sol)
+
+pragma solidity ^0.8.19;
+
+import {IERC165} from "./IERC165.sol";
+
+/**
+ * @title IERC721MultiMetadata (EIP-7160) optional EIP-721 multi-metadata extension
+ * @author 0xG
+ * @author mpeyfuss
+ * @dev See https://eips.ethereum.org/EIPS/eip-7160 and https://eips.ethereum.org/EIPS/eip-721
+ *      Note: the ERC-165 identifier for this interface is 0x06e1bc5b.
+ */
+interface IERC7160 is IERC165 {
```

Comment:
Called the interface after the EIP but would it be possible to rename it to IERC721MultiMetadata? What's is the best practice?